Library/Homebrew/formula_creator.rb

```diff
@@ -57,6 +50,7 @@ def head?
     end
 
     def generate!
```

Comment:
```suggestion
    def generate_path
```
or something feels more appropriate if this is now returning a `path`.

Reply:
It creates the formula file on disk and returns the path to it. Does `write_formula!` sound good enough?

Reply:
Yeh, that sounds good.

Reply:
Fixed.